sci-biology/ants/ants-9999.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit cmake-utils git-r3
+
+DESCRIPTION="Advanced Normalitazion Tools for neuroimaging"
+HOMEPAGE="http://stnava.github.io/ANTs/"
+SRC_URI=""
+EGIT_REPO_URI="git://github.com/stnava/ANTs.git"
+
+SLOT="0"
+LICENSE="BSD"
+KEYWORDS=""
+
+DEPEND="sci-libs/itk"
+RDEPEND="${DEPEND}"
+
+src_install() {
+	pwd
+	ls
+	cd "${P}_build/ANTS-build"
+	emake DESTDIR="${D}" install
```

Comment:
that is strange, navigating to that dir after the build fails and running make install gives me this:

```
-- Installing: /usr/lib/libl_antsAffineInitializer.a
-- Installing: /usr/bin/antsAffineInitializer
-- Installing: /usr/lib/libl_SurfaceBasedSmoothing.a
-- Installing: /usr/bin/SurfaceBasedSmoothing
-- Installing: /usr/lib/libl_CreateJacobianDeterminantImage.a
-- Installing: /usr/bin/CreateJacobianDeterminantImage
-- Installing: /usr/lib/libl_ThresholdImage.a
-- Installing: /usr/bin/ThresholdImage
-- Installing: /usr/lib/libl_ResampleImage.a
-- Installing: /usr/bin/ResampleImage
-- Installing: /usr/lib/libl_sccan.a
-- Installing: /usr/bin/sccan
-- Installing: /usr/lib/libl_N4BiasFieldCorrection.a
-- Installing: /usr/bin/N4BiasFieldCorrection
-- Installing: /usr/lib/libl_N3BiasFieldCorrection.a
-- Installing: /usr/bin/N3BiasFieldCorrection
-- Installing: /usr/lib/libl_KellyKapowski.a
-- Installing: /usr/bin/KellyKapowski
-- Installing: /usr/lib/libl_antsRegistration.a
-- Installing: /usr/bin/antsRegistration
-- Installing: /usr/lib/libl_antsMotionCorrStats.a
-- Installing: /usr/bin/antsMotionCorrStats
-- Installing: /usr/lib/libl_antsMotionCorr.a
-- Installing: /usr/bin/antsMotionCorr
-- Installing: /usr/lib/libl_antsApplyTransforms.a
-- Installing: /usr/bin/antsApplyTransforms
-- Installing: /usr/lib/libl_LabelGeometryMeasures.a
-- Installing: /usr/bin/LabelGeometryMeasures
-- Installing: /usr/lib/libl_LabelClustersUniquely.a
-- Installing: /usr/bin/LabelClustersUniquely
-- Installing: /usr/lib/libl_Atropos.a
-- Installing: /usr/bin/Atropos
-- Installing: /usr/lib/libl_AverageImages.a
-- Installing: /usr/bin/AverageImages
-- Installing: /usr/lib/libl_StackSlices.a
-- Installing: /usr/bin/StackSlices
-- Installing: /usr/lib/libl_AverageAffineTransform.a
-- Installing: /usr/bin/AverageAffineTransform
-- Installing: /usr/lib/libl_AverageAffineTransformNoRigid.a
-- Installing: /usr/bin/AverageAffineTransformNoRigid
-- Installing: /usr/lib/libl_antsApplyTransformsToPoints.a
-- Installing: /usr/bin/antsApplyTransformsToPoints
-- Installing: /usr/lib/libl_antsAI.a
-- Installing: /usr/bin/antsAI
-- Installing: /usr/lib/libl_antsJointFusion.a
-- Installing: /usr/bin/antsJointFusion
-- Installing: /usr/lib/libl_ImageMath.a
-- Installing: /usr/bin/ImageMath
-- Installing: /usr/lib/libl_iMath.a
-- Installing: /usr/bin/iMath
-- Installing: /usr/lib/libl_ANTS.a
-- Installing: /usr/bin/ANTS
-- Installing: /usr/lib/libl_ANTSJacobian.a
-- Installing: /usr/bin/ANTSJacobian
-- Installing: /usr/lib/libl_PrintHeader.a
-- Installing: /usr/bin/PrintHeader
-- Installing: /usr/lib/libl_ResetDirection.a
-- Installing: /usr/bin/ResetDirection
-- Installing: /usr/lib/libl_ANTSUseLandmarkImagesToGetAffineTransform.a
-- Installing: /usr/bin/ANTSUseLandmarkImagesToGetAffineTransform
-- Installing: /usr/lib/libl_ANTSUseLandmarkImagesToGetBSplineDisplacementField.a
-- Installing: /usr/bin/ANTSUseLandmarkImagesToGetBSplineDisplacementField
-- Installing: /usr/lib/libl_ANTSUseDeformationFieldToGetAffineTransform.a
-- Installing: /usr/bin/ANTSUseDeformationFieldToGetAffineTransform
-- Installing: /usr/lib/libl_LaplacianThickness.a
-- Installing: /usr/bin/LaplacianThickness
-- Installing: /usr/lib/libl_SetOrigin.a
-- Installing: /usr/bin/SetOrigin
-- Installing: /usr/lib/libl_SetSpacing.a
-- Installing: /usr/bin/SetSpacing
-- Installing: /usr/lib/libl_SetDirectionByMatrix.a
-- Installing: /usr/bin/SetDirectionByMatrix
-- Installing: /usr/lib/libl_SurfaceCurvature.a
-- Installing: /usr/bin/SurfaceCurvature
-- Installing: /usr/lib/libl_ConvertScalarImageToRGB.a
-- Installing: /usr/bin/ConvertScalarImageToRGB
-- Installing: /usr/lib/libl_CreateWarpedGridImage.a
-- Installing: /usr/bin/CreateWarpedGridImage
-- Installing: /usr/lib/libl_MeasureImageSimilarity.a
-- Installing: /usr/bin/MeasureImageSimilarity
-- Installing: /usr/lib/libl_ConvertToJpg.a
-- Installing: /usr/bin/ConvertToJpg
-- Installing: /usr/lib/libl_ConvertImage.a
-- Installing: /usr/bin/ConvertImage
-- Installing: /usr/lib/libl_ConvertImagePixelType.a
-- Installing: /usr/bin/ConvertImagePixelType
-- Installing: /usr/lib/libl_ConvertInputImagePixelTypeToFloat.a
-- Installing: /usr/bin/ConvertInputImagePixelTypeToFloat
-- Installing: /usr/lib/libl_FitBSplineToPoints.a
-- Installing: /usr/bin/FitBSplineToPoints
-- Installing: /usr/lib/libl_AverageTensorImages.a
-- Installing: /usr/bin/AverageTensorImages
-- Installing: /usr/lib/libl_ImageSetStatistics.a
-- Installing: /usr/bin/ImageSetStatistics
-- Installing: /usr/lib/libl_MultiplyImages.a
-- Installing: /usr/bin/MultiplyImages
-- Installing: /usr/lib/libl_SmoothImage.a
-- Installing: /usr/bin/SmoothImage
-- Installing: /usr/lib/libl_ClusterImageStatistics.a
-- Installing: /usr/bin/ClusterImageStatistics
-- Installing: /usr/lib/libl_LabelOverlapMeasures.a
-- Installing: /usr/bin/LabelOverlapMeasures
-- Installing: /usr/lib/libl_LesionFilling.a
-- Installing: /usr/bin/LesionFilling
-- Installing: /usr/lib/libl_MeasureMinMaxMean.a
-- Installing: /usr/bin/MeasureMinMaxMean
-- Installing: /usr/lib/libl_WarpImageMultiTransform.a
-- Installing: /usr/bin/WarpImageMultiTransform
-- Installing: /usr/lib/libl_ComposeMultiTransform.a
-- Installing: /usr/bin/ComposeMultiTransform
-- Installing: /usr/lib/libl_MemoryTest.a
-- Installing: /usr/bin/MemoryTest
-- Installing: /usr/lib/libl_PermuteFlipImageOrientationAxes.a
-- Installing: /usr/bin/PermuteFlipImageOrientationAxes
-- Installing: /usr/lib/libl_ImageCompare.a
-- Installing: /usr/bin/ImageCompare
-- Installing: /usr/lib/libl_ResampleImageBySpacing.a
-- Installing: /usr/bin/ResampleImageBySpacing
-- Installing: /usr/lib/libl_CopyImageHeaderInformation.a
-- Installing: /usr/bin/CopyImageHeaderInformation
-- Installing: /usr/lib/libl_WarpTimeSeriesImageMultiTransform.a
-- Installing: /usr/bin/WarpTimeSeriesImageMultiTransform
-- Installing: /usr/lib/libl_ExtractSliceFromImage.a
-- Installing: /usr/bin/ExtractSliceFromImage
-- Installing: /usr/lib/libl_ExtractRegionFromImage.a
-- Installing: /usr/bin/ExtractRegionFromImage
-- Installing: /usr/lib/libl_ExtractRegionFromImageByMask.a
-- Installing: /usr/bin/ExtractRegionFromImageByMask
-- Installing: /usr/lib/libl_PasteImageIntoImage.a
-- Installing: /usr/bin/PasteImageIntoImage
-- Installing: /usr/lib/libl_TileImages.a
-- Installing: /usr/bin/TileImages
-- Installing: /usr/lib/libl_CreateTiledMosaic.a
-- Installing: /usr/bin/CreateTiledMosaic
-- Installing: /usr/lib/libl_CreateImage.a
-- Installing: /usr/bin/CreateImage
-- Installing: /usr/lib/libl_DenoiseImage.a
-- Installing: /usr/bin/DenoiseImage
-- Installing: /usr/lib/libl_WarpTensorImageMultiTransform.a
-- Installing: /usr/bin/WarpTensorImageMultiTransform
-- Installing: /usr/lib/libl_ReorientTensorImage.a
-- Installing: /usr/bin/ReorientTensorImage
-- Installing: /usr/lib/libl_RebaseTensorImage.a
-- Installing: /usr/bin/RebaseTensorImage
-- Installing: /usr/lib/libl_KellySlater.a
-- Installing: /usr/bin/KellySlater
-- Installing: /usr/lib/libl_CreateDTICohort.a
-- Installing: /usr/bin/CreateDTICohort
-- Installing: /usr/lib/libl_antsAlignOrigin.a
-- Installing: /usr/bin/antsAlignOrigin
-- Installing: /usr/lib/libl_simpleSynRegistration.a
-- Installing: /usr/bin/simpleSynRegistration
-- Installing: /usr/lib/libl_CompositeTransformUtil.a
-- Installing: /usr/bin/CompositeTransformUtil
-- Installing: /usr/lib/libl_antsMotionCorrDiffusionDirection.a
-- Installing: /usr/bin/antsMotionCorrDiffusionDirection
-- Installing: /usr/lib/libl_antsSliceRegularizedRegistration.a
-- Installing: /usr/bin/antsSliceRegularizedRegistration
-- Installing: /usr/lib/libl_CreateDisplacementField.a
-- Installing: /usr/bin/CreateDisplacementField
-- Installing: /usr/lib/libl_ANTSIntegrateVectorField.a
-- Installing: /usr/bin/ANTSIntegrateVectorField
-- Installing: /usr/lib/libl_ANTSIntegrateVelocityField.a
-- Installing: /usr/bin/ANTSIntegrateVelocityField
-- Installing: /usr/lib/libl_ConvertTransformFile.a
-- Installing: /usr/bin/ConvertTransformFile
-- Installing: /usr/lib/libl_compareTwoTransforms.a
-- Installing: /usr/bin/compareTwoTransforms
-- Installing: /usr/lib/libl_TimeSCCAN.a
-- Installing: /usr/bin/TimeSCCAN
-- Installing: /usr/lib/libl_TextureCooccurrenceFeatures.a
-- Installing: /usr/bin/TextureCooccurrenceFeatures
-- Installing: /usr/lib/libl_TextureRunLengthFeatures.a
-- Installing: /usr/bin/TextureRunLengthFeatures
-- Installing: /usr/lib/libl_ImageIntensityStatistics.a
-- Installing: /usr/bin/ImageIntensityStatistics
-- Installing: /usr/lib/libl_GetConnectedComponentsFeatureImages.a
-- Installing: /usr/bin/GetConnectedComponentsFeatureImages
-- Installing: /usr/lib/libl_DeNrrd.a
-- Installing: /usr/bin/DeNrrd
-- Installing: /usr/lib/libl_antsTransformInfo.a
-- Installing: /usr/bin/antsTransformInfo
-- Installing: /usr/lib/libl_antsUtilitiesTesting.a
-- Installing: /usr/bin/antsUtilitiesTesting
-- Installing: /usr/bin/jointfusion
-- Installing: /usr/bin/sa
-- Installing: /usr/bin/bl
```

And indeed I can see those files were placed there. How can I make this work via portage? also, how could I remove the files which I now installed? a bit many to rm y hand :-/